server/app.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
```

Comment:
use import instead of require

Reply:
Node.js doesn't support ES6's import yet. So that's why used require